### DISCUSSION TITLE
Fixed our Nebari with tip from troubleshooting doc!

### DISCUSSION THREAD

**Discussion by rsignell** [nebari-dev/nebari#2368] [General]
The gateway clusters on our Nebari deployment weren't starting, giving a 500 error.  
I was panicking because I was supposed to teach a class using that deployment in a few minutes time -- no time to raise an issue and wait for a reply. 

I thought: "Maybe there is a troubleshooting section in the docs?"

So I googled "nebari troubleshooting" and sure enough, I found:
https://www.nebari.dev/docs/troubleshooting/#new-dask-gateway-clusters-lead-to-500-internal-server-error

Yay!  So I went to k9s, deleted those two pods, they regenerated, and life was good!    

Phew, class saved!     Awesome job Nebari doc team!

**Comment by mwengren** [comment]
Thanks @rsignell!  I just had the same issue trying to get a Dask notebook working for a demo in my Nebari deployment and this post (and the linked troubleshooting doc) saved my butt!  CTRL + D to the rescue!

I'd be curious why the Dask Gateway pods failed in the first place, and also how often this is likely to be necessary in a long-running Nebari deployment (hopefully not too often), but those are probably questions for another day.  Thanks again for posting this.

**Reply by rsignell** [reply]
Glad it was helpful @mwengren !
